KaggleTorrent/main.py

```diff
@@ -57,13 +63,47 @@ def main():
     db_engine = db_connection_handler.create_sqlalchemy_engine()
     print("Connected to the database")
 
-    if command == 'generate':
-
+    # CHECK USER VARIABLES
+    proceed = None
+
+    # Check db emptiness
+    if database_exists(db_engine.url):
+        if command == 'init':
+            print(f'Database {config.db_name} already exist. Provide database name that does not exist.',
+                  file=sys.stderr)
+            proceed = False
+        if command == 'refresh':
+            print(f'Database {config.db_name} already exist. This operation will reinitialize the current database')
+            print('and populate it with provided MetaKaggle version.')
+            ans = input(f'Are you sure to re-initialize {config.db_name} database? [yes]\n')
+            if ans.lower() == 'yes':
+                proceed = True
+            else:
+                proceed = False
+    else:
+        if command == 'refresh':
+            print(f'Database {config.db_name} does not exist. Provide database name that contain a MetaKaggle version.',
+                  file=sys.stderr)
+            proceed = False
+        proceed = True
+
+    # Check download folder emptiness when init
+    has_data = next(Path(config.nb_archive_path).iterdir(), None)
+    if (has_data is not None) & (command == 'init'):
+        print(f'Download folder {config.nb_archive_path} is not empty.', file=sys.stderr)
+        print('Provide an empty folder for download notebooks.', file=sys.stderr)
```

Comment:
Qui sarebbe meglio:

"Please, provide the path to an empty folder to store downloaded notebooks."